src/main/java/com/mealmentor/enterprise/MealMentorController.java

```diff
@@ -50,7 +55,10 @@ public MealItem createMealItem(@RequestBody MealItem mealItem) throws Exception
         try{
             newMealItem = mealPlanService.save(mealItem);
         } catch (Exception e) {
-            //TODO ADD LOGGING
+            logger.error("Cannot create a meal item");
+            e.printStackTrace();
+            throw(e);
+            // not necessary but if you'd like, redirect user to the error page here.
```

Comment:
added logging in the try-catch block.
I also created an error page (not necessary - just a suggestion). If you'd like, you can redirect to that page whenever something goes wrong (like this case) and you'd like to let the user know.